portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,45 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>OliviaA: Home</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="http://www.w3schools.com/lib/w3.css">
   </head>
-  <body>
+
+<header>
+    <h1>Home</h1>
+    <p> Hey there, Welcome to my Site!</p>
+</header>
+<div class="menu">
+    <nav>
+        <a href="gallery.html"> Photos </a>
+         <!--
```

Comment:
I'd remove the commented section links until you are ready to add them.

Reply:
will do!